src/flatland/ordered/map.clj

```diff
@@ -21,15 +19,7 @@
                          )
            (java.util Map Map$Entry)))
 
-;; We could use compile-if technique here, but hoping to avoid
```

Comment:
Could you please explain to me what this has to do with Clojure < 1.6? I'm not familiar at all and would like to learn

Reply:
@slipset The implementation fell back on `hash-unordered-coll` when it existed and something else if that didn't exist. `hash-unordered-coll` was introduced in Clojure 1.6.